.eslintrc.js

```diff
@@ -38,7 +38,8 @@ module.exports = {
           ['@contexts', './src/contexts'],
           ['@pages', './src/pages'],
           ['@utils', './src/utils'],
-          ['@components', './src/components']
+          ['@components', './src/components'],
+          ['@constants', './src/constants']
```

Comment:
I've been just adding these to the list as I encounter them. Not directly relevant to any changes.